### PR TITLE
Project-Happy-Thoughts - Tina Bruce

### PR DESCRIPTION
**Pull request by brucette**
Link to my deployed project:

https://dapper-croquembouche-c4091f.netlify.app/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Happy Thoughts
```
Comment (Tanmark) [inline README.md:1]: Just have to start by complementing your readme, they are usually skipped over. But yours is good!

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,74 @@
...
 export const App = () => {
+  const [thoughtList, setThoughtList] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [newThought, setNewThought] = useState('');
+
+  // Function to get the thoughts currently in the API:
+  const fetchThoughts = () => {
+    setLoading(true);
+    fetch('https://happy-thoughts-technigo.herokuapp.com/thoughts')
+      .then((res) => res.json())
```
Comment (Tanmark) [inline code/src/App.js:14]: Cleare and neat code throughout the app.js. Very easy to follow with your comments, love that.

File: code/src/components/ThoughtForm.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+export const ThoughtForm = ({ onFormSubmit, newThought, handleNewThought }) => {
+  return (
+    <section className="form">
+      <form onSubmit={onFormSubmit}>
+        <h4>What&apos;s making you happy right now?</h4>
+        <textarea
+          placeholder="Enter your message here.."
```
Comment (Tanmark) [inline code/src/components/ThoughtForm.js:9]: The &apos - nice! found that too after many red squiggly lines...

File: code/src/components/ThoughtList.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { SingleThought } from 'components/SingleThought';
+
+export const ThoughtList = ({ loading, thoughtList }) => {
+  if (loading) {
+    return <h2>Loading in progress...</h2>
+  }
+
+  // Map through all the thoughts currently in the API & display them in a div:
+  return (
```
Comment (Tanmark) [inline code/src/components/ThoughtList.js:10]: Good explanation of map

File: code/src/components/SingleThought.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        <div>
+          <button
+            hearts={singleThoughthearts}
+            type="button"
+            onClick={handleLikeBtnClick}
+            className={singleThoughthearts === 0 ? 'heartBtn' : 'heartBtn pink'}>
+            ❤️
+          </button>
+          &nbsp; x {singleThoughthearts}&nbsp;
+        </div>
```
Comment (Tanmark) [inline code/src/components/SingleThought.js:32]: Does this give the heart count? Send me a line, haven't seen it before (I think).

File: code/src/index.css [hunk 6]
```diff
@@ -1,13 +1,90 @@
...
+  background-color: #F2F0F0;
+  border-radius: 50%;
+  border: 0px;
+}
+
+.pink {
+  background-color: #FFAEAD;
+}
+
+.submitBtn:hover, .heartBtn:hover {
```
Comment (Tanmark) [inline code/src/index.css:67]: This is a nice touch, same thing with the heart that also expands

**Review by Tanmark — COMMENTED** [review]
Overall all nice coding with clear descriptions  which makes it easy to follow. I see no deadends, forgotten code or bits that not really come together. The site works in mobile, tablet and desktop. All functions of the brief are there. Very solid performance, be proud!

File: code/src/components/SingleThought.js (hunk 5 above)
Comment (brucette) [reply code/src/components/SingleThought.js:32]: This is just to insert a space between the heart button and the x, and between the x and the actual count :)

**Pull request closed without merging**